flashlight/autograd/backend/cpu/Conv2D.cpp

```diff
@@ -42,9 +42,26 @@ Variable conv2d(
     int py,
     int dx,
     int dy,
-    int groups) {
+    int groups,
+    bool useMixedPrecision,
+    const ConvPerfAuditPtr audit) {
+  if (useMixedPrecision) {
+    throw std::runtime_error("Half precision is not supported in CPU.");
```

Comment:
We can handle this case similarly by checking the `input.dtype()` and throwing if it `== af::dtype::f16`.